### PR TITLE
19th - one page app

### PR DESCRIPTION
**Pull request by GeekRuthie**
Here's the draft. Comments welcome.  Nothing fancy here, just a trick I've used some.

Issue #51

### CODE REVIEW

File: pending/19-one-page-apps.pod [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+ajax '/messages' => sub {
+   my @messages = (
+      'Boo! I betcha I scared you, ha-ha!',
+      'A girl has no name, but she still has a list.',
+      'Space, the final frontier.',
+      '',
+      '<h1>Really big text</h1>',
+   );
+   return to_json({ message => @messages[int(rand(5))] });
```
Comment (cromedome) [inline pending/19-one-page-apps.pod:42]: Alternatively, you could write this as `send_as JSON => { message => @messages[int(rand(5))] };`

**Comment by cromedome** [comment]
I like this! Short, sweet, and to the point. Really useful contribution! :+1:

File: pending/19-one-page-apps.pod (hunk 1 above)
Comment (GeekRuthie) [reply pending/19-one-page-apps.pod]: I've made that change!

**Comment by cromedome** [comment]
Merged, thanks!

**Pull request closed without merging**